resources/icons/16px/subtract-join.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M4 8C4 6.14529 4.72785 4.40776 6.0056 3.10116C5.68075 3.03483 5.34444 3 5 3C2.23858 3 0 5.23858 0 8C0 10.7614 2.23858 13 5 13C5.34301 13 5.67796 12.9655 6.00155 12.8997C4.73646 11.6111 4 9.86556 4 8ZM11 3C13.7614 3 16 5.23858 16 8C16 10.7614 13.7614 13 11 13C10.6575 13 10.323 12.9656 9.99981 12.8999C11.2643 11.6096 12 9.8646 12 8C12 6.13491 11.2639 4.38952 9.9997 3.10035C10.3223 3.03451 10.6571 3 11 3ZM12.001 4.127C12.6516 5.30526 13 6.63054 13 8C13 9.25507 12.7074 10.473 12.1617 11.568L12 11.873L12.1553 11.8306C13.7402 11.3533 14.9106 9.91921 14.9951 8.19964L15 8C15 6.19252 13.8012 4.66505 12.1553 4.16936L12.001 4.127Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M10 7.99946C10 6.36419 9.21496 4.91226 8.0012 4L7.91804 4.06199C6.75033 4.97727 6 6.40075 6 7.99946C6 9.6352 6.78547 11.0875 8.0012 11.9989C9.21496 11.0867 10 9.63474 10 7.99946ZM7.88898 5.48438L7.999 5.35446L8.07238 5.43734C8.66295 6.14432 9 7.04336 9 7.99946L8.99303 8.23722C8.94207 9.10408 8.61374 9.91352 8.07238 10.5616L8 10.6425L7.92844 10.5616C7.33732 9.85508 7 8.95602 7 7.99946C7 7.06486 7.3219 6.18449 7.88898 5.48438Z" fill="#5F6B7C"/>
+</svg>
```

Comment:
In addition to @ryannono 's [comment](https://github.com/palantir/blueprint/pull/7132#issuecomment-2551615976) on unioning the path, can we also omit the `width`, `height` `fill` and `fill/clip-rule` attributes from the final SVG?
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M6.0056 3.10116C4.72785 4.40776 4 6.14529 4 8C4 9.86556 4.73646 11.6111 6.00155 12.8997C5.67796 12.9655 5.34301 13 5 13C2.23858 13 0 10.7614 0 8C0 5.23858 2.23858 3 5 3C5.34444 3 5.68075 3.03483 6.0056 3.10116ZM16 8C16 5.23858 13.7614 3 11 3C10.6571 3 10.3223 3.03451 9.9997 3.10035C11.2639 4.38952 12 6.13491 12 8C12 9.8646 11.2643 11.6096 9.99981 12.8999C10.323 12.9656 10.6575 13 11 13C13.7614 13 16 10.7614 16 8ZM13 8C13 6.63054 12.6516 5.30526 12.001 4.127L12.1553 4.16936C13.8012 4.66505 15 6.19252 15 8L14.9951 8.19964C14.9106 9.91921 13.7402 11.3533 12.1553 11.8306L12 11.873L12.1617 11.568C12.7074 10.473 13 9.25507 13 8ZM8.0012 4C9.21496 4.91226 10 6.36419 10 7.99946C10 9.63474 9.21496 11.0867 8.0012 11.9989C6.78547 11.0875 6 9.6352 6 7.99946C6 6.40075 6.75033 4.97727 7.91804 4.06199L8.0012 4ZM7.999 5.35446L7.88898 5.48438C7.3219 6.18449 7 7.06486 7 7.99946C7 8.95602 7.33732 9.85508 7.92844 10.5616L8 10.6425L8.07238 10.5616C8.61374 9.91352 8.94207 9.10408 8.99303 8.23722L9 7.99946C9 7.04336 8.66295 6.14432 8.07238 5.43734L7.999 5.35446Z"/>
</svg>
```